hw/arm/stellaris.c

```diff
@@ -1337,6 +1395,19 @@ static void lm3s6965evb_init(MachineState *machine)
     stellaris_init(machine, &stellaris_boards[1]);
 }
 
+static void vul_suc_init(MachineState *machine)
+{
+    VulSucMachineState *s = VUL_SUC_MACHINE(machine);
+
+    for(int i = 0; i < ARRAY_SIZE(s->sockdmas); i++) {
```

Comment:
nit:
```suggestion
    for (int i = 0; i < ARRAY_SIZE(s->sockdmas); i++) {
```
(There are more similar occurrences.)